addons/portal/static/src/scss/portal.scss

```diff
@@ -81,6 +81,11 @@ a, .btn-link {
     }
 }
 
+// close button must be on right side (behavior of button.close before BS5 mig)
+button.btn-close {
+    float: right;
+}
+
```

Comment:
@tde-banana-odoo I'm not much used to css & assets, I guess there is a risk of side-effects, but I still think this fix is useful directly in the portal css :thinking:.  Think it's still safe or should be encapsulated under a portal specific element/class ?

(can be easily spotted when going on a SO in the portal, and adding `&message=sign_ok` in the url)

Reply:
cc: @rfr-odoo

Reply:
Ok, that's a dumb fix :rofl:
Thanks for looking (but it seems that there is quite a lot of `alert-dismissable` left in odoo code :eyes:)